.github/workflows/ci.yml

```diff
@@ -189,14 +189,20 @@ jobs:
           - pre-command:
             briefcase-run-prefix:
             briefcase-run-args:
+            setup-python: true
 
           - backend: macOS
             runs-on: macos-12
 
+          # We use a fixed Ubuntu version rather than `-latest` because at some point,
+          # `-latest` will be updated, but it will be a soft changeover, which would cause
+          # the system Python version to become inconsistent from run to run.
           - backend: linux
-            runs-on: ubuntu-latest
-            pre-command: "sudo apt-get update -y && sudo apt-get install -y python3-gi python3-gi-cairo gir1.2-gtk-3.0 python3-dev libgirepository1.0-dev libcairo2-dev pkg-config libfuse2"
+            runs-on: ubuntu-22.04
+            # The package list should be the same as in tutorial-0.rst.
```

Comment:
This list also exists in Toga README, and the BeeWare tutorial (docs.beeware.org) step 0. AFAICT, all three should be in sync.

Reply:
OK, I've made the README reference the Toga tutorial, and created https://github.com/beeware/beeware/pull/196 for the BeeWare tutorial.